params/defaults.go

```diff
@@ -22,13 +22,13 @@ const (
 	DefaultGas = 180000
 
 	// WhisperMinimumPoW amount of work for Whisper message to be added to sending queue
-	WhisperMinimumPoW = 0.001
+	WhisperMinimumPoW = 0.000002
```

Comment:
Would it make sense to add a comment here about us enforcing a minimum?
https://github.com/status-im/status-go/blob/a21f8a39bc4b786999011f88f519bf518fbfa299/params/config.go#L84-L85

Reply:
added

Reply:
That's nice, but I meant in `status-go/params/config.go`. Where most people would look for explanations of options.

Reply:
added in the right place as well :)

Reply:
Cheers!